server/test/mock-notion-api.js

```diff
@@ -0,0 +1,719 @@
+class NotionAPIMocks {
+
+  requirements_db_1 = {
+    object: "list",
+    results: [
+      {
+        object: "page",
+        id: "e72e315f-4e40-4a70-81ef-f491d88b38f1",
+        created_time: "2023-03-04T15:06:00.000Z",
+        last_edited_time: "2023-03-04T15:14:00.000Z",
+        created_by: {
+          object: "user",
+          id: "08b376c9-4f60-4f57-9e9e-132c4191ba57",
+        },
+        last_edited_by: {
+          object: "user",
+          id: "08b376c9-4f60-4f57-9e9e-132c4191ba57",
+        },
+        cover: null,
+        icon: null,
+        parent: {
+          type: "database_id",
+          database_id: "b9c7195b-d9fa-414a-97ee-704f503a0e9f",
+        },
+        archived: false,
+        properties: {
+          "Created by": {
+            id: "%3FGva",
+            type: "created_by",
+            created_by: {
+              object: "user",
+              id: "08b376c9-4f60-4f57-9e9e-132c4191ba57",
+              name: "David",
+              avatar_url: "https://s3-us-west-2.amazonaws.com/public.notion-static.com/626e4886-6f95-47ee-8788-9672a6673ef4/Cosmic.jpg",
+              type: "person",
+              person: {
+                email: "d.maranto@mail.utoronto.ca",
+              },
+            },
+          },
+          Parent: {
+            id: "%40KmU",
+            type: "relation",
+            relation: [
+              {
+                id: "8e49640e-4461-43eb-a0e8-9c85eacbb411",
+              },
+            ],
+            has_more: false,
+          },
+          "Last Edited": {
+            id: "%40U%60y",
+            type: "last_edited_time",
+            last_edited_time: "2023-03-04T15:14:00.000Z",
+          },
+          Qualifier: {
+            id: "L%5D%60q",
+            type: "relation",
+            relation: [
+              {
+                id: "795fff9e-776a-4408-87e6-576d8a9af133",
+              },
+            ],
+            has_more: false,
+          },
+          Collection: {
+            id: "OEtd",
+            type: "multi_select",
+            multi_select: [
+            ],
+          },
+          "Type (legacy)": {
+            id: "R%3AvZ",
+            type: "select",
+            select: null,
+          },
+          Child: {
+            id: "R%5C~k",
+            type: "relation",
+            relation: [
+            ],
+            has_more: false,
+          },
+          "🏁 Verification Plan": {
+            id: "RnoY",
+            type: "relation",
+            relation: [
+            ],
+            has_more: false,
+          },
+          "🍄 Type": {
+            id: "Z~z%5D",
+            type: "relation",
+            relation: [
+              {
+                id: "d484af0c-f3f9-41c9-8afc-543693ec608d",
+              },
+            ],
+            has_more: false,
+          },
```

Comment:
`Type` is no longer a property of the database, Should this be removed?

Reply:
Oops, updated!